NitroxClient/MonoBehaviours/MultiplayerButton.cs

```diff
@@ -0,0 +1,234 @@
+﻿using NitroxModel.Helper;
+using NitroxClient.MonoBehaviours;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.UI;
+using UWE;
+using NitroxClient.Communication;
+
+namespace NitroxClient.MonoBehaviours
```

Comment:
Yeah maybe put all of this stuff in a new namespace NitroxClient.MonoBehaviours.Gui.MainMenu